### PR TITLE
Forward fix for templates + views

### PR DESCRIPTION
**Pull request by Chillee**
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #126320
* __->__ #127446

cc @voznesenskym @penguinwu @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @wenzhe-nrv @jiayisunx @peterbell10 @ipiszy @yf225 @chenyang78 @kadeng @muchulee8 @ColinPeppler @amjames @desertfire @chauhang

### CODE REVIEW

**Review by eellison — APPROVED** [review]

File: torch/_inductor/codegen/simd.py [hunk 1]
```diff
@@ -1403,8 +1403,9 @@ def codegen_template(
                 for node in [template_node, *epilogue_nodes]:
                     node.mark_run()
             partial_code = render()
-            for node in epilogue_nodes:
-                node.codegen(kernel.split_and_set_ranges(node.get_ranges()))
+            with kernel.set_subgraph_body("<STORE_OUTPUT>"):
+                for node in epilogue_nodes:
+                    node.codegen(kernel.split_and_set_ranges(node.get_ranges()))
```
Comment (ezyang) [inline torch/_inductor/codegen/simd.py:1408]: OOC, how did you figure this out in the end?

Comment (Chillee) [reply torch/_inductor/codegen/simd.py:1408]: It's just how the templating works 🤔 When you call `node.codegen` on the epilogue nodes, it implicitly writes to global context (including `.body` potentially). Previously in my testing it just wasn't writing to `V.kernel.body` in this code, but the stack trace was indicating that it did here.

**Pull request closed without merging**